src/app/shared/context.service.spec.ts

```diff
@@ -182,9 +184,10 @@ describe('Context Service:', () => {
     });
 
     // when
-    contextService.changeContext(navigation).subscribe(val => {});
+    contextService.changeContext(navigation).subscribe(() => {});
```

Comment:
what is the point of this? you pass nothing in , return nothing, and do nothing, right?

Reply:
Well... I'm asking myself the same question. @corinnekrych might know?